rca/project_styleguide/templates/patterns/molecules/pagination/pagination.html

```diff
@@ -1,11 +1,11 @@
-{% load util_tags %}
+{% load util_tags querystring %}
 
-{% if subpages.paginator.num_pages > 1 %}
+{% if paginator_page.paginator.num_pages > 1 %}
```

Comment:
might need to update the `pagination.yaml` file as well with this variable change